cmd/containerd-shim-runhcs-v1/options/runhcs.proto

```diff
@@ -62,7 +62,12 @@ message Options {
 
 	// GPUVHDPath is the path to the gpu vhd to add to the uvm
 	// when a container requests a gpu
-	string GPUVHDPath = 10; 
+	string GPUVHDPath = 10;
+
+        // scale_cpu_limits_to_sandbox indicates that container CPU limits should
+        // be adjusted to account for the difference in number of cores between the
+        // host and UVM.
+        bool scale_cpu_limits_to_sandbox = 11;
```

Comment:
I wonder if we should rename this to something like "adjust_cpu_limits_to_sandbox" to make it clearer that this is a special case? I'm not tied to either way, but just an idea.

Reply:
I don't really think either term is any clearer than the other, so prefer to just leave it as is. :)